include/CL/opencl.hpp

```diff
@@ -2466,93 +2467,10 @@ class Device : public detail::Wrapper<cl_device_id>
     }
 #endif // #if CL_HPP_TARGET_OPENCL_VERSION >= 210
 
-#if CL_HPP_TARGET_OPENCL_VERSION >= 120
-    //! \brief Wrapper for clCreateSubDevices().
-    cl_int createSubDevices(
-        const cl_device_partition_property * properties,
-        vector<Device>* devices)
```

Comment:
It looks like this version of the function (the OpenCL 1.2 version that uses `cl_device_partition_property`, not `cl_device_partition_property_ext`) was erroneously removed.  We need to keep it.  Note, I think it can stay here, in the class definition, because it doesn't need to get the platform to query extension pointers.

Reply:
that's right.
Just fixed.
Thanks!